playkit/src/main/java/com/kaltura/playkit/PlayerEngineWrapper.java

```diff
@@ -185,6 +185,11 @@ public BaseTrack getLastSelectedTrack(int renderType) {
         return playerEngine.getLastSelectedTrack(renderType);
     }
 
+    @Override
+    public List<EventStream> getEventStreams() {
```

Comment:
@GouravSna  do we have to support IMA and DAI Engine wrapper

Reply:
For the time, I don't think so because this info is coming in dash manifest for SSAI implementation.
But yes, this should be support for HLS media.

For HLS, the events are signaled using the EXT-X-DATERANGE tag and a combination of the EXT-X-CUE-OUT and EXT-X-CUE-IN tags, while for DASH they are signaled in DASH Event Messages. The EXT-X-DATERANGE and EXT-X-CUE-OUT plus EXT-X-CUE-IN tags in HLS present different ways of signaling similar information.

Reply:
I think, for this version. We don't need it. This </EventStream> info is actually giving the leverage to mimic the behaviour of IMADAI. For IMA, it is not required.